code/cryptography/src/morse_cipher/morse_code_generator.c

```diff
@@ -1,22 +1,25 @@
 /* Part of Cosmos by OpenGenus Foundation */
 //C implementation of morse code generator
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 
-void encrypt(char* msg){
-    int i, size;
-    char morse_code[1000] = "";
+void 
+encrypt(char* msg)
+{
+    int i, size = strlen(msg);
```

Comment:
Is it possible you could move `i` closer to the for loop? (Maybe move `size` as well?)